cmd/statusd/utils.go

```diff
@@ -267,6 +268,8 @@ func testGetDefaultConfig(t *testing.T) bool {
 
 func testResetChainData(t *testing.T) bool {
 	resetChainDataResponse := common.APIResponse{}
+
+	//TODO(influx6): Should we apply the copy of chain directory and restore after test here?
```

Comment:
Do we need this `testResetChainData` test at all? We have another test that tests this functionality. 

Generally, we should tests bindings with mocks.

Reply:
@tiabc I am in agreement here, i believe the test we have running is more than enough

Reply:
> Generally, we should tests bindings with mocks.

I meant that all C bindings should be tested against mocks to just make sure that they call proper methods in a proper order. But it was just a general comment, we will fix all binding tests in a different ticket.

Regarding this test, if we can fix it so that it does not affect other tests, then let's do it. Otherwise, let's verify that `statusAPI.ResetChainDataAsync()` is tested and if so, I think we can safely remove `testResetChainData` test.

Reply:
@adambabik After re-reading this a few times, I gather what you mean is to replace `testResetChainData` usage of `statusAPI.ResetChainData` to `statusAPI.ResetChainDataAsync` but do we want to run `ResetChainData` in asyn mode without validating `sync status` using the blocking `SyncPoll`?

Reply:
Hmm, for now I will comment `testResetChainData` then add test for `StatusBackend.ResetChainDataAsync` in `geth/api`.

Reply:
@adambabik From what I can see `ResetChainData` is well tested (see https://github.com/status-im/status-go/blob/refactoring/blockchain-sync%23246/geth/api/backend_test.go#L485) and even `statusAPI.ResetChainData` and `statusAPI.ResetChainDataAync` (see https://github.com/status-im/status-go/blob/refactoring/blockchain-sync%23246/geth/api/backend_test.go#L312-L317, https://github.com/status-im/status-go/blob/refactoring/blockchain-sync%23246/geth/api/api_test.go#L89-L94).

From my point of view, I really dont think we need `testResetChainData` in cmd. @adambabik @tiabc

Reply:
Thanks for checking that! If we have tested underlying methods, I also think this test is obsolete. Let's wait for @tiabc confirmation.

Reply:
Agree.